packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -266,10 +266,28 @@ addGraphQLResolvers({
           rating: 0
         }
       })
-    }
+    },
+    async UsersReadPostsOfTargetUser(root: void, { userId, targetUserId, limit = 20 }: { userId: string, targetUserId: string, limit: number }, context: ResolverContext) {
```

Comment:
Hm, now that I think about it, we should prefer to define new query resolver using `defineQuery`.  Then you can define the query inline instead of using `addGraphQLQuery` as on L284.

Reply:
can we skip for now? surely this is Fine Enough? Want to ship at some point!

Reply:
In my experience such changes rarely get integrated later, though some do get rolled up into a big batch PR that takes somewhere between multiple hours and multiple days and carries a non-trivial chance of breaking things.  (Also this fix should only take like 30 seconds)